src/adapter.js

```diff
@@ -14,6 +14,29 @@ var formatError = function (error) {
   return message
 }
 
+var processAssertionError = function (error_) {
+
+  var error
+
+  if (window.Mocha && error_.hasOwnProperty('showDiff')) {
+
+    error = {
+      name: error_.name,
+      message: error_.message,
+      showDiff: error_.showDiff
+    }
+
+    if (error.showDiff) {
+      error.actual = window.Mocha.utils.stringify(error_.actual)
+      error.expected = window.Mocha.utils.stringify(error_.expected)
```

Comment:
This crash when `actual` or `expected ` has a circular reference: https://github.com/mochajs/mocha/issues/4910. This happens when actual is a DOM element rendered by React. 

@roadhump what was the use case for this logic?